db/migrate/20220326103529_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ]  Here also, Comment should reference the `author_id` which is an alias of the `users` table. Kindly use the hint I gave you in the Post reference to implement this so that your comments table will have a reference column `author_id` instead of `user_id` as seen in the ERD.